S2VX.Game/Editor/UserInterface/NoteSnapDivisorDisplay.cs

```diff
@@ -0,0 +1,44 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+
+namespace S2VX.Game.Editor.UserInterface {
+    public class NoteSnapDivisorDisplay : CompositeDrawable {
+        [Resolved]
+        private S2VXEditor Editor { get; set; }
+
+        private TextFlowContainer TxtNoteSnapDivisor { get; set; }
+
+        public Anchor TextAnchor { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void Load() =>
+            InternalChildren = new Drawable[]
+            {
+                TxtNoteSnapDivisor = new TextFlowContainer(s => s.Font = new FontUsage("default", Editor.DrawWidth / 40, "500")) {
+                    RelativeSizeAxes = Axes.Both,
+                    RelativePositionAxes = Axes.Both,
+                    TextAnchor = TextAnchor,
+                }
+            };
+
+        protected override void Update() {
+            if (Editor.SnapDivisor == 0) {
+                TxtNoteSnapDivisor.Text = "Snap Divisor: Free";
+            } else {
+                TxtNoteSnapDivisor.Text = $"Snap Divisor: {Editor.SnapDivisor}";
```

Comment:
I think it would have more meaning if the snap divisor value showed the fraction that would be snapped, e.g. snap display text 1 for rounding to whole numbers, 0.5 for rounding to halves, 0.25 for rounding to quarters, etc.